controllers/cart.js

```diff
@@ -0,0 +1,64 @@
+const { products, carts, orders } = require('../storage');
+const crypto = require('crypto');
+
+function renderCart(id) {
+  if (carts.find((item) => item.userId === id)) {
+    return;
+  } else {
+    const cart = {
+      id: crypto.randomUUID(),
+      userId: id,
+      products: [],
+    };
+    carts.push(cart);
+  }
+}
+
+function addProductToCart(id, product) {
+  const thisProduct = products.find((item) => item.id === +product);
+  const thisCart = carts.find((item) => item.userId === id);
+  const arrOfProducts = thisCart.products;
```

Comment:
lines 20, 21, 22 can be replaced with a single line `thisCart.products.push(thisProduct)`